chains/solana/contracts/programs/cctp-token-pool/src/lib.rs

```diff
@@ -70,6 +73,24 @@ pub mod cctp_token_pool {
         ctx.accounts.state.config.accept_ownership()
     }
 
+    pub fn set_fund_manager(ctx: Context<SetConfig>, fund_manager: Pubkey) -> Result<()> {
+        ctx.accounts.state.fund_manager = fund_manager;
+        Ok(())
+    }
+
+    pub fn set_minimum_signer_funds(ctx: Context<SetConfig>, amount: u64) -> Result<()> {
+        ctx.accounts.state.minimum_signer_funds = amount;
+        Ok(())
+    }
+
+    pub fn set_fund_reclaim_destination(
+        ctx: Context<SetConfig>,
+        fund_reclaim_destination: Pubkey,
+    ) -> Result<()> {
+        ctx.accounts.state.fund_reclaim_destination = fund_reclaim_destination;
+        Ok(())
+    }
```

Comment:
Please emit an event in each of these (it can be a specific event for each or the same event with all fields in all of them, as you prefer)